src/core/interp.c

```diff
@@ -5632,6 +5633,11 @@ void MVM_interp_run(MVMThreadContext *tc, void (*initial_invoke)(MVMThreadContex
                 cur_op += 20;
                 goto NEXT;
             }
+            OP(cpucores): {
```

Comment:
fixing, there seem to be a few other cases like this. do you guys usually clean up cosmetic stuff  as part of another commit when you come across it, or do it separately? also: do you squash your PRs?